app/Config/core.php

```diff
@@ -172,9 +172,36 @@
  * the cake shell command: cake schema create Sessions
  *
  */
-	Configure::write('Session', array(
-		'defaults' => 'php'
-	));
+
+    // AWS DynamoDB sessions
+    if (getenv('COMANAGE_REGISTRY_DYNAMODB_REGION') and
+        getenv('COMANAGE_REGISTRY_DYNAMODB_PHPSESSIONS_TABLE') and
+        getenv('COMANAGE_REGISTRY_DYNAMODB_PHPSESSIONS_ACCESSKEY') and
+        getenv('COMANAGE_REGISTRY_DYNAMODB_PHPSESSIONS_SECRETACCESSKEY')) {
```

Comment:
Is there a reason to use `and` instead of `&&` here?

See eg [this discussion](https://stackoverflow.com/questions/2803321/and-vs-as-operator).